Update public document (#10843)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender-endpoint/linux-preferences.md

- automatic_definition_update_enabled

> [!NOTE]

> > For the mdatp_managed.json to take effect, restart of the `mdatp` deamon is required only in cases when `cloud-diagnostic` or `log-rotation-parameters` configs are updated.

Reviewer: Reword as follows:

No restart of mdatp deamon is required for changes to **most** configs in mdatp_managed.json to take effect.
**Exception**: Following configs will require deamon restart to take effect:
- cloud-diagnostic
- log-rotation-parameters

Author: Done

- automatic_definition_update_enabled

> [!NOTE]

> > No restart of mdatp deamon is required for changes to most configs in mdatp_managed.json to take effect.
>   **Exception:** Following configs will require deamon restart to take effect:

Reviewer 2: ```suggestion
> No restart of mdatp daemon is required for changes to _most_ configurations in mdatp_managed.json to take effect.
  **Exception:** The following configurations require a daemon restart to take effect:
```
